CHANGELOG.md

```diff
@@ -11,6 +17,8 @@ x.x.x Release notes (yyyy-MM-dd)
 * Add support for the `[d]` modifier on string comparison operators to perform
   diacritic-insensitive comparisons.
 * Explicitly mark `[[RLMRealm alloc] init]` as unavailable.
+* Don't unnecessarily advance the Realm transaction when resolving a
```

Comment:
Yes, that's the commit that prompted me to write this entry, but you're right that it doesn't apply. Removed.